code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1216,6 +1216,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 			H.throw_alert("nutrition", /obj/screen/alert/hungry)
 		if(0 to NUTRITION_LEVEL_STARVING)
 			H.throw_alert("nutrition", /obj/screen/alert/starving)
+			H.adjustBruteLoss(2)
```

Comment:
someone remind me, how often does this proc?

Reply:
once every tick (2 seconds), provided that you have a non-failing stomach.

in other words, you'll be critted in 100 seconds if you don't get medical attention or something to eat.

Reply:
bad